prow_config.yaml

```diff
@@ -54,4 +52,14 @@ workflows:
       - xgboost-job/*
     kwargs:
       build_and_apply: false
-      config_path: https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_istio_dex.yaml
\ No newline at end of file
+      config_path: https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_istio_dex.yaml
```

Comment:
em. I know kubeflow minimum distribution is still being discussed, but I hope to keep it until we figure it out If this is not a problem at this moment.